webrtc/libjingle/xmpp/boshsocket.cc

```diff
@@ -155,7 +156,15 @@ namespace buzz {
 #ifdef FEATURE_ENABLE_SSL
     if (tls_ != buzz::TLS_DISABLED) {
       first_socket = rtc::SSLAdapter::Create(first_socket);
+// For non-production builds, relax certificate requirements. This will allow the use
+// of a debugging proxy, for example.
+#ifndef _PRODUCT
```

Comment:
@willkelleher : depends if we build them with PSY_PRODUCT cmake variable set (psy_product -> _PRODUCT define is in psyplatform\psy_settings.cmake).

Arguably, we should be doing this, but maybe some build script needs to be tweaked.